api/v1alpha1/groupversion_info.go

```diff
@@ -4,7 +4,7 @@
 // Package v1alpha1 contains API Schema definitions for the kamaji v1alpha1 API group
 // +kubebuilder:object:generate=true
 // +groupName=kamaji.clastix.io
-//nolint
+// nolint
```

Comment:
May I ask you the reason in this fix?

Reply:
Sorry, this may be a problem with my IDE formatting, I will fall back.